app/src/androidMain/kotlin/de/westnordost/streetcomplete/data/user/achievements/AchievementDefinitionsModule.kt

```diff
@@ -292,11 +292,11 @@ val links = listOf(
         R.string.link_osmapp_description
     ),
     Link(
-        "organic_maps",
-        "https://organicmaps.app/",
-        "Organic Maps",
+        "co_maps",
+        "https://comaps.app/",
+        "CoMaps",
         LinkCategory.MAPS,
-        Res.drawable.link_organic_maps,
+        Res.drawable.link_co_maps,
         R.string.link_organic_maps_description
```

Comment:
I guess that we keep this now a bit confusing name rather than migrate translations manually?

(maybe we should have list of cases like this and at some point rename them?)